src/main/java/mods/railcraft/common/carts/EntityTunnelBore.java

```diff
@@ -873,9 +870,8 @@ protected boolean mineBlock(BlockPos targetPos, BlockRailBase.EnumRailDirection
                 }
             }
         }
-        WorldPlugin.setBlockToAir(world, targetPos);
```

Comment:
When does this happen now?

Reply:
It is handled in `WorldPlugin#playerRemoveBlock`, a method originally used by the destruction enchantment of the crowbar.

Reply:
So before the drops are calculated? Isn't this a problem?

Reply:
block removal should not be bound with dropping... does cancelling drop event only in forge prevent block breaking?

Reply:
Restored.